API/Services/SeriesService.cs

```diff
@@ -520,6 +520,11 @@ public async Task<SeriesDetailDto> GetSeriesDetail(int seriesId, int userId)
             .SelectMany(v => v.Chapters.Where(c => !c.IsSpecial))
             .OrderBy(c => float.Parse(c.Number), new ChapterSortComparer());
 
+        // When there's chapters without a volume number revert to chapter sorting only as opposed to volume then chapter
+        if (storylineChapters.Any()) {
+            retChapters = retChapters.OrderBy(c => float.Parse(c.Number), new ChapterSortComparer());
```

Comment:
Let's move the new comparator to a private member to avoid recreating each API call.

Reply:
That's a good idea. We can even go further and implement the Comparer Default static property pattern.